ql/src/test/results/clientnegative/msck_repair_2.q.out

```diff
@@ -21,4 +21,4 @@ POSTHOOK: Output: default@repairtable
 PREHOOK: query: MSCK TABLE default.repairtable
 PREHOOK: type: MSCK
 PREHOOK: Output: default@repairtable
-FAILED: Execution Error, return code 1 from org.apache.hadoop.hive.ql.ddl.DDLTask
+#### A masked pattern was here ####
```

Comment:
Real exception is 
`FAILED: Execution Error, return code 40000 from org.apache.hadoop.hive.ql.ddl.DDLTask. org.apache.hadoop.hive.metastore.api.MetastoreException: MSCK finds a file rather than a directory when it searches for file:/Users/tarak/IdeaProjects/tarak-Apache-Hive/itests/qtest/target/localfs/warehouse/repairtable/p1=c/datafile`

which is getting masked as it contained location of file